pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -106,6 +106,7 @@ const (
 			table_name varchar(261) NOT NULL,
 			engine_id int NOT NULL,
 			status tinyint unsigned DEFAULT 30,
+			commit_ts bigint NOT NULL DEFAULT 0,
```

Comment:
Bigint ranges from -9223372036854775808 to 9223372036854775807, it may cause TSO(uint64) overflow.